README.md

```diff
@@ -1,6 +1,11 @@
 README for Mbed TLS
 ===================
 
+License
```

Comment:
We don't need a full license section at the top. I suggest moving this section just before “Contributing”, and adding a short sentence “It is distributed under the Apache-2.0 license.” to the introductory paragraph.

Reply:
OK, I'll move this to just before Contributing. I don't see the point in duplicating information in the introduction - that's just more stuff to maintain.